org.eclipse.xtext.ui.shared/plugin.properties

```diff
@@ -2,6 +2,7 @@ OpenModelElementCommand.name=Open Model Element
 OpenModelElementCommand.description=Open the declaration of an Xtext element
 XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
+XtextNature.description=Adds an icon decoration to projects with Xtext enabled.
```

Comment:
shouldn't this be something like "Convert the project to an Xtext project"? I mean, adding a nature is more than adding an icon decoration